Codes/coursetable.php

```diff
@@ -21,8 +21,25 @@
 
         <?php
         require_once __DIR__ . '/connection/connect.php';
+        $id = $_SESSION['id'];
+        $uname = $_SESSION['name'];
         if (isset($_GET['course'])) {
             $cd = $_GET['course'];
+            $sql = "SELECT id FROM courses WHERE course_code='$cd'";
+            $result = $conn->query($sql);
+            if($row = $result->fetch_assoc())
+            {
```

Comment:
In this line you should first check that number of rows in the result should be one